app/src/main/java/com/arctouch/io/outdoorsychallenge/OutdoorsyApplication.kt

```diff
@@ -1,7 +1,6 @@
 package com.arctouch.io.outdoorsychallenge
 
 import android.app.Application
-import androidx.databinding.library.BuildConfig
```

Comment:
wrong reference of BuildConfig removed. Due to that, `BuildConfig.DEBUG` was always false and Timber couldn't be initialized.